packages/core/src/index.ts

```diff
@@ -125,6 +125,13 @@ const initialize = async ({
   // Only attempt to check for CodePush bundle URL in release mode. Otherwise CodePush will throw an exception.
   // https://docs.microsoft.com/en-us/appcenter/distribution/codepush/react-native#plugin-configuration-ios
   if (!__DEV__) {
+    const bundleJs =
+      await NativeModules.EmbraceManager.getDefaultJavaScriptBundlePath();
+
+    if (bundleJs) {
+      await NativeModules.EmbraceManager.setJavaScriptBundlePath(bundleJs);
```

Comment:
I await this since I want Codepush to be the last to be applied

Reply:
In the case of Codepush being implemented does this mean we'd be computing the bundle ID twice?

An alternative might be to stick this default behaviour in the else case here when codepush isn't setup: https://github.com/embrace-io/embrace-react-native-sdk/blob/9e999556384b761774cc678d232c07992d27e0b0/packages/core/ios/RNEmbrace/EmbraceManager.swift#L224, that could also address this being an ios-only special case